cdap-proto/src/test/java/co/cask/cdap/proto/id/EntityIdTest.java

```diff
@@ -115,8 +127,16 @@ public void testInvalidId() {
   @Test
   @Ignore
   public void testPrintToString() {
-    for (ElementId id : ids) {
-      System.out.println(id.toString());
+    for (EntityId id : ids) {
+      System.out.println(id.getEntity() + ": " + id.toString());
```

Comment:
I suppose it does make sense to verify the toString() output, will add a test case.